decidim-proposals/app/commands/decidim/proposals/admin/assign_proposals_to_valuator.rb

```diff
@@ -32,6 +33,10 @@ def call
 
         attr_reader :form
 
+        def send_email
+          ProposalsValuatorMailer.notify_proposals_valuator(form.valuator_role.user, form.current_user, form.proposals).deliver_later
+        end
```

Comment:
@andreslucena @entantoencuanto I'm also wondering if instead of creating a mailer, I should use the Events Manager like this: https://github.com/decidim/decidim/blob/develop/decidim-proposals/app/commands/decidim/proposals/admin/create_proposal_note.rb#L51

Reply:
I think it's a good idea to use the events mechanism to generate the notification and from what I see its use is quite widespread.